codec/codec.go

```diff
@@ -24,17 +24,19 @@ type (
 		InterfaceRegistry() types.InterfaceRegistry
 
 		// GetMsgAnySigners returns the signers of the given message encoded in a protobuf Any
-		// as well as the decoded google.golang.org/protobuf/proto.Message that was used to
-		// extract the signers so that this can be used in other contexts.
-		GetMsgAnySigners(msg *types.Any) ([][]byte, protov2.Message, error)
-
-		// GetMsgV2Signers returns the signers of the given message.
-		GetMsgV2Signers(msg protov2.Message) ([][]byte, error)
-
-		// GetMsgV1Signers returns the signers of the given message plus the
-		// decoded google.golang.org/protobuf/proto.Message that was used to extract the
-		// signers so that this can be used in other contexts.
-		GetMsgV1Signers(msg proto.Message) ([][]byte, protov2.Message, error)
+		// as well as the decoded protoreflect.Message that was used to extract the
+		// signers so that this can be used in other context where proto reflection
+		// is needed.
+		GetMsgAnySigners(msg *types.Any) ([][]byte, protoreflect.Message, error)
+
+		// GetMsgSigners returns the signers of the given message plus the
+		// decoded protoreflect.Message that was used to extract the
+		// signers so that this can be used in other context where proto reflection
+		// is needed.
+		GetMsgSigners(msg proto.Message) ([][]byte, protoreflect.Message, error)
```

Comment:
need a changelog entry for these changes

Reply:
If I get a concept ACK, I'll take care of the remaining items

Reply:
this makes it more clear and clarifies what is needed

Reply:
CHANGELOG entry added